apps/frontend/vitest.config.ts

```diff
@@ -5,8 +5,8 @@ export default defineConfig({
   plugins: [react()],
   test: {
     environment: 'jsdom',
-    globals: true,
     setupFiles: ['./vitest.setup.ts'],
+    include: ['./libs/**/*.spec.ts'],
```

Comment:
```suggestion
   exclude: ['**/node_modules/**', '**/playwright/**'],
```

playwright 폴더를 제외시키는 건 어떨까요???